lib/mailgun_catcher/event.rb

```diff
@@ -11,7 +11,7 @@ def notify_bugsnag
     end
 
     def error_message
-      "[#{severity}] [#{reason}] #{recipient}: #{delivery_status}"
+      "[#{severity}] [#{reason}] #{recipient}"
```

Comment:
I do not want you to change the error message, we just need to change to grouping rule.

https://github.com/infinum/rails-infinum-mailgun_catcher/blob/master/lib/mailgun_catcher/event_error.rb#L19

Reply:
Aaaah, ok, sorry :D